Index: stjerm-0.16/src/Makefile.am
===================================================================
--- stjerm-0.16.orig/src/Makefile.am	2012-07-20 17:51:24.000000000 -0400
+++ stjerm-0.16/src/Makefile.am	2013-04-15 21:08:14.519957302 -0400
@@ -11,7 +11,8 @@
 LIBS = @glib_LIBS@ @LIBS@\
        @gtk_LIBS@\
        @gthread_LIBS@\
-       @vte_LIBS@
+       @vte_LIBS@\
+       -lXinerama
 	     
 AM_CFLAGS = @CFLAGS@\
 	    @glib_CFLAGS@\
